### ISSUE TITLE
Map Statuses

### ISSUE THREAD

**Issue by DebugOk** [DeltaV-Station/Delta-v#366]
# Implemented with maintainer
* Tortuga - Velcro
* Arena - Velcro
* Hive - Velcro
* Pebble - Field Command
* Lighthouse - Deltanedas
* Glacier - Deltanedas
* Byoin - Field Command
* Elegance - Gray
* Terra - SirrRequiem

# Implemented, but no maintainer
* Edge - (Unclaimed)
* Shoko - (Unclaimed)
* Asterisk - (Unclaimed)
* Chibi - (Unclaimed)
* Division - (Unclaimed)

# Implemented but not in rotation
* Submarine - Rose/LSD (Needs overhaul)
* Micro - (Unclaimed) (Needs SM)
* Hammurabi - Velcro (Needs overhaul)

# In progress
* Ovni - Deltanedas (Requires full rework) 
* Anvil - Gray (New Map)
* Acadamy - Calis (New Map)

**Comment by DebugOk** [comment]
Oh also literally everything in /Shuttles/, but thats already covered by #346

**Comment by IamVelcroboy** [comment]
You could probably add Submarine, a map currently under review

**Comment by DebugOk** [comment]
> You could probably add Submarine, a map currently under review

Not really valid, this only applies to old maps currently. I'll add it when its merged

**Comment by deltanedas** [comment]
is anyone reviving glacier :trollface:

**Comment by IamVelcroboy** [comment]
> is anyone reviving glacier :trollface:

Not until if/when we get proper planet maps/atmos.

**Comment by Colin-Tel** [comment]
Asterisk has entered the fray

**Comment by IamVelcroboy** [comment]
Carolyn is maintaining Pebble now.

**Comment by ColdDead** [comment]
With the cards in general, swap the unnecessary medic records at the outpost to the crew monitor computers, which make more sense, I would like to use the outpost more but the thing with the computers doesn't work out well

**Comment by Axionyxx** [comment]
This should be done via the wiki instead imo
